.circleci/config.yml

```diff
@@ -704,6 +709,7 @@ workflows:
     jobs:
       - api-bigquery-test
       - puppeteer-e2e-test:
+          parallel_num: 1
```

Comment:
Only one e2e test is running in nightly job so parallelism: 1 makes sense.

Reply:
I would probably run it with the same parallelism here - is there a reason to make it slower?

Reply:
`registration.spec.ts` is the only test that runs nightly. More machines to run one test is actually failing the job from jest error because there isn't any test to run on other machines.